vendor/vendor.json

```diff
@@ -24,6 +24,13 @@
 			"revision": "97646858c46433e4afb3432ad28c12e968efa298",
 			"revisionTime": "2017-08-22T15:24:03Z"
 		},
+		{
+			"checksumSHA1": "M1jObMAw/YV+Rc5PccfmJuuxLa0=",
+			"origin": "github.com/chrisccoulson/go-tpm",
```

Comment:
Please check if this package is available in Debian and Fedora. Our packaging policy requires that.

Reply:
This is for the spike only